dev-util/xdelta/xdelta-3.1.0.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A binary diff tool that uses the VCDIFF (RFC 3284) format and compression"
```

Comment:
Can you shrink this line to something <= 70 chars, like, for instance:
```
SUMMARY="A binary diff tool that uses the VCDIFF (RFC 3284) format"
```
This is because SUMMARY is used by HaikuDepot, which needs to show it in a big list.
(See https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#summary for more details.)
Thanks!